src/components/emails/WelcomeEmail.tsx

```diff
@@ -0,0 +1,122 @@
+import {
+  Body,
+  Button,
+  Container,
+  Head,
+  Hr,
+  Html,
+  Link,
+  Preview,
+  Section,
+  Text,
+} from "@react-email/components";
+import * as React from "react";
+import Logo from "@/components/Logo/index";
+
+const baseUrl = process.env.VERCEL_URL
+  ? `https://${process.env.VERCEL_URL}`
+  : "";
+
+interface StripeWelcomeEmailProps {
+  firstName: string;
+  programName: string;
+}
+
+export const StripeWelcomeEmail: React.FC<StripeWelcomeEmailProps> = ({
+  firstName,
+  programName,
+}) => (
+  <Html>
+    <Head />
+    <Preview>
+      You&apos;ve been accepted into St. Christopher&apos;s Trucker Relief Fund!
+    </Preview>
+    <Body style={main}>
+      <Container style={container}>
+        <Section style={box}>
+          <Logo
```

Comment:
Right now, your email doesn't build if you use `pnpm run email-dev`
According to react-email docs, SVGs are not well supported in email clients, which is what our logo component uses. My bad on that one, but I added a logo.png to the repo. Reference that and use the react-email image component instead (unless Next.js starts complaining and wants you to use theirs)